corehq/apps/analytics/tests/test_partner_analytics_utils.py

```diff
@@ -175,12 +166,7 @@ def delete_user(user):
         ]
         for form in forms:
             elastic_form = transform_xform_for_elasticsearch(form.to_json())
-            send_to_elasticsearch('forms', elastic_form)
-
-        cls.es.indices.refresh(USER_INDEX_INFO.alias)
-        cls.es.indices.refresh(XFORM_INDEX_INFO.alias)
-        cls.addClassCleanup(ensure_index_deleted, USER_INDEX_INFO.alias)
-        cls.addClassCleanup(ensure_index_deleted, XFORM_INDEX_INFO.alias)
+            form_adapter.index(elastic_form, refresh=True)
```

Comment:
This will refresh after every form, isn't it better to do that outside the loop?

Reply:
It would be better but will it matter much given that the document count would be really low in tests?